buffer/src/test/java/io/netty/buffer/ByteBufStreamTest.java

```diff
@@ -247,4 +251,44 @@ public void testRead() throws Exception {
         buf2.release();
         in2.close();
     }
+
+    @Test
+    public void testReadLineLengthRespected() throws Exception {
```

Comment:
will create 2 separated test case for these

Reply:
sure why not